### ISSUE TITLE
video plays inside popups : cannot tab to CC when JAWS is activated

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#9]
When a wet video player is inside a popup, it seems you cannot tab to the CC from the progress bar
